pom.xml

```diff
@@ -348,11 +367,17 @@
                     <fail>true</fail>
                     <cvssScoreThreshold>7.0</cvssScoreThreshold>
                     <excludeCoordinates>
+                        <exlude>
+                            <!-- Trello card for this exclude: https://trello.com/c/ahkwEM74/1720-babbage-elasticsearch-library-multiple-vulnerabilities-s2 -->
```

Comment:
I don't have access to this board, could you explain why is this being excluded, please? Also, the link looks the same as the one below (in line 377), I'm just mentioning this just in case it's a typo.

Reply:
ossindex will complain about vulnerabilities here and fail the build with an error. so to ignore those libraries for scans they are excluded here. I don't have access either but the first two entries are about eleasticsearch vulnerabilities. Latest elastic search version  is way higher  then the version ONS currently using.